static/mint.html

```diff
@@ -0,0 +1,439 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mint | Ubiquity Dollar</title>
+    <link rel="stylesheet" href="style/style.css" />
+  </head>
+  <body class="w-screen overflow-x-hidden min-h-screen drawer">
+    <main class="container mx-auto flex max-w-full lg:w-[80%] flex-col justify-start items-center py-4 px-4 lg:px-5 gap-12 overflow-x-hidden drawer-content">
```

Comment:
Styling in class name / Tailwind is not used within our UIs, tho I am not sure if it matters as styling will be made in a future PR

Reply:
Tailwind should not be used.